src/test/java/ikysil/training/dao/OrderDaoTest.java

```diff
@@ -0,0 +1,49 @@
+package ikysil.training.dao;
+
+import ikysil.training.dao.impl.InMemoryOrderDao;
+import ikysil.training.ws.api.v1.dto.OrderDto;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.List;
+
+import static org.junit.Assert.*;
+
+public class OrderDaoTest {
+
+    private OrderDao dao = new InMemoryOrderDao();
+
+    private OrderDto order1 = new OrderDto("1144", "twenty one", "none", "3129", "firstuser@gmail.com");
+    private OrderDto order2 = new OrderDto("3257", "twenty two", "none", "4024", "seconduser@gmail.com");
+
+    @Before
+    public void before(){
+        dao.saveOrder(order1);
+        dao.saveOrder(order2);
+    }
+
+    @Test
+    public void test(){
```

Comment:
you could add separate tests for . e.g.:
1 `saveOrder`
2 `updateOrder`, update should replace old value
3 `deleteOrder`
4 `findAll` works if empty and returns empty list.
5 test inserting null values
6 saving duplicates